Gemfile

```diff
@@ -25,22 +25,22 @@ def location_for(place_or_version, fake_version = nil)
   end
 end
 
-group :development, :test do
+group :development do
   gem 'rake',                    :require => false
-  gem 'mocha', '~>0.10.5',       :require => false
-  gem 'puppetlabs_spec_helper',  :require => false
-  gem 'puppet-lint',             :require => false
   gem 'rspec', '~>2.14.0',       :require => false
+  gem 'puppet-lint',             :require => false
+  gem 'puppetlabs_spec_helper',  :require => false
+  gem 'puppet_facts',            :require => false
+  gem 'mocha', '~>0.10.5',       :require => false
 end
-
-beaker_version = ENV['BEAKER_VERSION'] || '~> 2.2'
 group :system_tests do
+  beaker_version = ENV['BEAKER_VERSION'] || '~> 2.2'
   if beaker_version
     gem 'beaker', *location_for(beaker_version)
   else
-    gem 'beaker',                :require => false, :platforms => :ruby
+    gem 'beaker', :require => false, :platforms => :ruby
```

Comment:
It seems that this code path is not necessary, is it @cyberious ?  To reach it, I need to intentionally set `ENV['BEAKER_VERSION']` to an empty string prior to installing gems, since `beaker_version` defaults to `~> 2.2` when it's not set..

Is this something we need to support for some reason in Linux / OSX?  (With `:platforms => :ruby` I don't believe that beaker will be installed ever on JRuby or Windows)

/cc @ferventcoder